app/services/api/v1/attendance/sync_service.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Attendance
+      class SyncService
+        include ActiveModel::Validations
+        validates_presence_of :date, :school_id
+        validate :validate_date
+
+        def initialize(params)
+          @params = params
+        end
+
+        def sync
+          return false unless valid?
+
+          find_attendances &&
+          set_result
+        end
+
+        attr_reader :result
+
+        private
+
+        def find_attendances
+          attendances ||= ::Attendance.where("school_id = ? AND date > ?", school_id, date)
+          @attendances = attendances.each_with_object({}) do |c, h|
```

Comment:
Instead of iterating over attendances can we group attendances by standard in sql query itself ?